x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/utils.ts

```diff
@@ -18,3 +18,24 @@ export const parsedPidOrEntityIdParameter = (parameters: {
     entity_id: parameters?.entityId?.[0] ?? '',
   };
 };
+
+const UNIT_TO_MILLISECONDS = (value: number) => ({
+  h: (): number => value * 60 * 60 * 1000,
+  m: (): number => value * 60 * 1000,
+  s: (): number => value * 1000,
+});
+
+const convertToMilliseconds = (value: number, unit: 'h' | 'm' | 's'): number =>
+  UNIT_TO_MILLISECONDS(value)[unit]();
+
+const EXECUTE_TIMEOUT_REGEX = /^\d+(?=(h|m|s){1}$)/;
+export const validateUnitOfTime = (value: string): boolean => EXECUTE_TIMEOUT_REGEX.test(value);
+
+export const parsedTimeoutInMilliseconds = (timeout?: string): number => {
+  const timeoutMatch = timeout?.trim().match(EXECUTE_TIMEOUT_REGEX);
+  if (!timeoutMatch) {
```

Comment:
if no timeout given default it to `2h` @paul-tavares

Reply:
Thats ok from the UI side, but this needs to also happen on the API

Reply:
a0447966d5cbf00c46a710f10484a896eacd2a15

Reply:
I updated this to 4hrs and removed the default value on the UI side. 70e3500519096e65f4cdc07740821f850b67ba8f